bin/create-scxa-analytics-schema.sh

```diff
@@ -313,7 +314,7 @@ curl -X POST -H 'Content-type:application/json' --data-binary '{
     "runtimeLib": true,
     "class": "uk.co.flax.biosolr.solr.update.processor.OntologyUpdateProcessorFactory",
     "annotationField": "ontology_annotation",
-    "ontologyURI": "file:///srv/gxa/scatlas.owl",
+    "ontologyURI": "'$SCXA_ONTOLOGY'",
```

Comment:
I don’t think you want the extra single quotes.

Reply:
It's because we're in single quotes mode - see also the CORE variable at line 292

Reply:
But doesn’t that mean that the expanded text will be e.g. `"ontologyURI": "'file:///srv/gxa/scatlas.owl'"`?

Reply:
Nope, the double quotes are literal, but the singles are just stopping and restarting the single quoted string I think. It's clear with syntax highlighting in Vim, and I checked it works

Reply:
I tried this:

`foo.sh`:
```bash
#!/usr/bin/env bash
FOO=bar
echo "'$FOO'"
```

And then:
```bash
$ ./foo.sh 
'bar'
```

I think the expanded text has single quotes but whatever is using it as an argument is trimming them, since they’re not present in `configoverlay.json` in Solr. Anyway, it just caught my eye.

Reply:
What you're missing is that the whole lot is in single quotes from the end of line 310. A more accurate reproduction is:

```
> FOO=bar
> echo '"'$FOO'"'
"bar"
```

Reply:
Ah! You’re totally right! Sorry for all the back and forth, I missed that.